ajax/libs/framework7/package.json

```diff
@@ -30,16 +30,15 @@
   ],
   "author": "Vladimir Kharlampidi",
   "license": "MIT",
-  "homepage": "http://www.idangero.us/framework7/",
+  "homepage": "https://framework7.io",
   "autoupdate": {
     "source": "git",
-    "target": "git://github.com/nolimits4web/Framework7.git",
+    "target": "git://github.com/framework7io/framework7.git",
     "fileMap": [
       {
-        "basePath": "dist",
+        "basePath": "packages/core",
```

Comment:
Not sure that should we also host `framework7-react.js`, `framework7-react.min.js`, `framework7-react.min.js.map` ...etc from https://github.com/framework7io/framework7/tree/master/packages/react.
As I know, React is the front-end framework, also here is an example for framework7-react: 
https://github.com/framework7io/framework7-react-static-template/blob/master/index.html#L18
Also, I think [vue](https://github.com/framework7io/framework7/tree/master/packages/vue) can be hosted, too. https://github.com/framework7io/framework7-template-vue-simple/blob/master/www/index.html#L339

What do you think?
Thank you.

Reply:
Lets ask authors opinion, @nolimits4web Would you like to answer @sufuf3 question?